packages/react/src/hooks/useAgent.ts

```diff
@@ -11,61 +11,60 @@ import { useParticipantAttributes } from './useParticipantAttributes';
 /**
  * @beta
  */
-export type VoiceAssistantState =
+export type AgentState =
   | 'disconnected'
   | 'connecting'
   | 'initializing'
   | 'listening'
   | 'thinking'
-  | 'speaking';
+  | 'speaking'
+  | string;
```

Comment:
this change allows users to define their own agent states not included in our default enumeration and still take advantage of the `useAgent` hook

Reply:
@lukasIO I changed this back too to keep it simple, but I'm wondering:

what happens if the backend sends a value not in this list, but the frontend expects only one of these five types? I'm assuming nothing too bad but I'm not really sure. We will definitely add more types in the future, in one way or another.

Reply:
that should be fine, others will just be ignored